src/tzkt/tzkt_api.py

```diff
@@ -65,7 +66,7 @@ def _request(self, path, **params):
         except requests.RequestException as e:
             raise TzKTApiError(e)
 
-        if response.status_code not in [200, 204]:
+        if response.status_code != HTTPStatus.OK:
```

Comment:
"ok" corresponds to two states? 200 and 204?

Reply:
It's only 200. `HTTPStatus.NO_CONTENT == 204`. see [docs](https://docs.python.org/3/library/http.html#http-status-codes).

Reply:
yes, it is way more expressive but why did you remove then the `HTTPStatus.NO_CONTENT == 204` was 204 not correct or irrelevant?

Reply:
see [comment](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/pull/405#issuecomment-813586548) above